source/test/F0.Minesweeper.Components.Tests/F0.Minesweeper.Components.Tests.csproj

```diff
@@ -7,6 +7,7 @@
 	<ItemGroup>
 		<PackageReference Include="bunit.web" Version="1.0.0-preview-01" />
 		<PackageReference Include="bunit.xunit" Version="1.0.0-preview-01" />
+		<PackageReference Include="Prism.Core" Version="8.0.0.1909" />
```

Comment:
do we need the `PackageReference` to _Prism.Core_ here as well ... I thing we depend on it transitively by referencing the `F0.Minesweeper.Components` project which has it already

Reply:
No, we don't need it. I uninstalled the package here.